sprites/Makefile

```diff
@@ -1,10 +1,14 @@
-PILL=Pill.json Pill.xpm
+TXTS=Big-Pill.txt Blinking-Terrified-Ghost.txt Pacman-Chomping-Down.txt Pacman-Chomping-Left.txt Pacman-Chomping-Right.txt Pacman-Chomping-Up.txt Pacman-Death.txt Pill.txt Red-Ghost-Down.txt Red-Ghost-Left.txt Red-Ghost-Right.txt Red-Ghost-Up.txt Red-Ghost-Win.txt Terrified-Ghost.txt
```

Comment:
I don't like the `TXTS` name very much. Maybe name it like `TXT_IMGS`? These are not ordinary text files, after all.